views/question.hbs

```diff
@@ -28,33 +28,63 @@
-          </form>
+          {{#if user}}
+            {{#each @root.question.voteHistory}}
+              {{#eq user @root.user.username}}
+                {{#eq current 1}}
+                  <button class="not-button"  id="button-up"><i class="glyphicon glyphicon-triangle-top up-voted" id="up"></i></button><br />
+                  <span class="nbOfVotes">{{@root.question.score}}</span><br />
+                  <form action="/question/{{@root.question.id}}/downupped" method="post">
+                    <input type="hidden" name="_method" value="PATCH">
+                    <button type="submit" class="not-button" id="button-down"><i class="glyphicon glyphicon-triangle-bottom not-voted" id="down"></i></button>
+                  </form>
+                {{/eq}}
+                {{#eq current -1}}
+                  <form action="/question/{{@root.question.id}}/updowned" method="post">
+                    <input type="hidden" name="_method" value="PATCH">
+                    <button type="submit" class="not-button"  id="button-up"><i class="glyphicon glyphicon-triangle-top not-voted" id="up"></i></button>
+                  </form>
+                  <span class="nbOfVotes">{{@root.question.score}}</span><br />
+                    <button type="submit" class="not-button" id="button-down"><i class="glyphicon glyphicon-triangle-bottom down-voted" id="down"></i></button>
+                  </form>
+                {{/eq}}
+              {{/eq}}
+            {{/each}}
+            {{#contains @root.question.users @root.user.username}}
+              {{else}}
+                <form action="/question/{{question.id}}/up" method="post">
+                  <input type="hidden" name="_method" value="PATCH">
+                  <button type="submit" class="not-button"  id="button-up"><i class="glyphicon glyphicon-triangle-top not-voted" id="up"></i></button>
+                </form>
+                <span class="nbOfVotes">{{question.score}}</span><br />
+                <form action="/question/{{question.id}}/down" method="post">
+                  <input type="hidden" name="_method" value="PATCH">
+                  <button type="submit" class="not-button" id="button-down"><i class="glyphicon glyphicon-triangle-bottom not-voted" id="down"></i></button>
+                </form>
+            {{/contains}}
+            {{else}}
+              <button class="not-button" id="button-up" onclick="alert('You must be logged in to vote.')"><i class="glyphicon glyphicon-triangle-top not-voted" id="up"></i></button><br>
+              <span class="nbOfVotes">{{@root.question.score}}</span><br />     
+              <button class="not-button" id="button-down" onclick="alert('You must be logged in to vote.')"><i class="glyphicon glyphicon-triangle-bottom not-voted" id="down"></i></button>       
```

Comment:
See above comment about `button` indentation and use of `i`.

Reply:
Also, JavaScript should not be included in the view, but rather an external script file.